### PR TITLE
Cross browser support

### PR DESCRIPTION
**Pull request by BhagyashreeBhoyar**

### CODE REVIEW

File: src/assets/css/dataGrid.css [hunk 1]
```diff
@@ -1249,6 +1262,8 @@ body,
     background: transparent;
     color: var(--orange-color);
```
Comment (Nikmahajan) [inline src/assets/css/dataGrid.css]: We can try naming a variable by its behavior, Do not use static name.
E.g: Here we are using  `--orange-color` it does'nt make sense to give it a color name specific you can either call it,
`--search-hover-color`, so whoever uses our application can define search hover color just by reading the name

File: src/assets/css/index.css [hunk 2]
```diff
@@ -293,13 +294,12 @@ input[type=number]::-webkit-outer-spin-button {
     font-weight: bold;
 }
 .footer {
-    background: var(--index-orange-color);
```
Comment (Nikmahajan) [inline src/assets/css/index.css]: same here for naming a variable.

File: src/components/routeComponents/StudentInformationGrid.jsx [hunk 3]
```diff
@@ -555,9 +566,11 @@ class StudentInformationGrid extends Component {
                 <div className="column-option display-mobile-none">
                   {/* <UploadOptInFile />*/}
                   <div className="column-option-configure display-inline">
-                    <Link to="/files" className="column-option-container text-decoration-none">
+                    <button className="column-option-container" onClick={this.redirectToFile}>
+                      {/* <Link to="/files" className="text-decoration-none">*/}
```
Comment (Nikmahajan) [inline src/components/routeComponents/StudentInformationGrid.jsx]: Remove commented code. it makes your code look dirty.

File: src/utils/http.js [hunk 4]
```diff
@@ -1,3 +1,4 @@
+import 'whatwg-fetch';
```
Comment (Nikmahajan) [inline src/utils/http.js]: Do we need this library???

File: webpack.config.js [hunk 5]
```diff
@@ -52,5 +53,10 @@ module.exports = {
       filename: './index.html',
       favicon: './src/assets/images/LOGO.png',
     }),
+    /* new webpack.DefinePlugin({
```
Comment (Nikmahajan) [inline webpack.config.js]: Please remove unnecessary code and imports which we are not using. Make sure to self Review your PR before assigning to anyone.

Comment (BhagyashreeBhoyar) [reply webpack.config.js]: Done.

File: src/components/routeComponents/StudentInformationGrid.jsx (hunk 3 above)
Comment (BhagyashreeBhoyar) [reply src/components/routeComponents/StudentInformationGrid.jsx]: Done.

File: src/assets/css/dataGrid.css (hunk 1 above)
Comment (BhagyashreeBhoyar) [reply src/assets/css/dataGrid.css]: I have renamed it by --theme-color.

File: src/assets/css/index.css (hunk 2 above)
Comment (BhagyashreeBhoyar) [reply src/assets/css/index.css]: I have renamed it by --default-background-color.

File: src/assets/css/dataGrid.css (hunk 1 above)
Comment (Nikmahajan) [reply src/assets/css/dataGrid.css]: If it is `--theme-color.`  then how the variable specifies its behaviour

File: src/utils/http.js (hunk 4 above)
Comment (BhagyashreeBhoyar) [reply src/utils/http.js]: It is used for fetch polyfill.

**Comment by pepcus-pratik** [comment]
@BhagyashreeBhoyar @Nikmahajan 
Can we merge this PR?

**Comment by Nikmahajan** [comment]
After some basic testing we can merge this PR.

**Pull request merged**